common/network/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
```

Comment:
What is the relationship between `core` and `common`? Networking feels like a "core" thing to me, so I'm a little confused here

Reply:
common --> library that has nothing to do with the specific domain, you could use it elsewhere
core --> library/module that has to do with the specific domain and it is shared by many features of your project
feature --> library/module that is specific for the domain and it is a specific feature
app/project --> the actual project that wires up features and everything

if we don't want to use this structure we can decide what to use, this is what I was going for

network in this case is a generic library that let you download a file you specify from the web and turn it into a string

Reply:
Ok, we can see how that goes for a bit. I feel like we've got a bit much going on at this "core/common" level but time will tell 😆